app/container-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eu
+
+bin='docker-gen'
+
+# run command if it is not starting with a "-" and is an executable in PATH
+if [ "${#}" -le 0 ] || \
+   [ "${1#-}" != "${1}" ] || \
+   [ -d "${1}" ] || \
+   ! command -v "${1}" > '/dev/null' 2>&1; then
+	entrypoint='true'
+fi
+
+exec ${entrypoint:+${bin:?}} "${@}"
+
+exit 0
```

Comment:
Not needed; `exec` cannot return unless it can't execute the command, in which case `exec`'s return value should be returned.

```suggestion
```